etl/steps/data/garden/tourism/2024-08-17/unwto_environment.meta.yml

```diff
@@ -0,0 +1,35 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Tourism
+    display:
+      numDecimalPlaces: 0
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+
+tables:
+  unwto_environment:
+    variables:
+      seea_tables:
+        title: Number of tables in the System of Environmental and Economic Account
+        unit: tables
+        description_short: Number of tables in the SEEA that track key environmental aspects of tourism, including water, energy, GHG emissions, and solid waste.
+
+      tsa_tables:
+        title: Number of tables in the Tourism Satellite Account
+        unit: tables
+        description_short: Number of tables in the TSA that track key economic aspects of tourism, including expenditure, production, and employment.
+
+      total_tables:
+       title: Total number of tables
+       unit: tables
+       description_short: Monitoring is expressed as the number of implemented tables across the Tourism Satellite Account and the System of Environmental and Economic Account. These tables track key economic and environmental aspects of tourism, including expenditure, production, employment, and resource flows.
```

Comment:
Hm, so looking at this description and the chart of this indicator, I (as a layperson) do not really understand what this is supposed to mean. 

I think this is mostly due to this metric being a bit abstract and not 1-to-1 corresponding with easily understandable things, but maybe you could elaborate a bit more on these tables (in the description_key or a dod). E.g. what is the TSA/ SEEA (an organisation? a method?), who is implementing/ tracking these, what are they used for? What does it really mean to have 1 table vs 8 tables here? 

Of course, if you don't have this information or there is no way to make it easily accessible to a wider audience, disregard this.

Reply:
thank you! I've added a more comprehensive description_key and slightly edited the description_short to make it a bit clearer!